Update README.md (#1)
the-turing-way/bookdash: wg-meeting-notes/2024-01-31_bookdash-wg-meeting.md

# The Turing Way Book Dash Working Group Meeting
## Date: 31 January 2024

## About this call

The Turing Way Book Dash Working Group meets on a monthly basis 

### Code of conduct

> \_The Turing Way\_ Code of conduct applies to this call. Take a moment to read this: [https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE\_OF\_CONDUCT.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE\_OF\_CONDUCT.md)

Author: ```suggestion
*The Turing Way* Code of conduct applies to this call. Take a moment to read this: [https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE\_OF\_CONDUCT.md](https://github.com/alan-turing-institute/the-turing-way/blob/master/CODE\_OF\_CONDUCT.md)
```

* For any question or concern please reach out to Malvika and Anne (msharan@turing.ac.uk, asteele@turing.ac.uk).

### Useful links:

   * Book Dash Chapter and Governance[https://github.com/the-turing-way/the-turing-way/issues/3231#issuecomment-1647578024](https://github.com/the-turing-way/the-turing-way/issues/3231#issuecomment-1647578024)
   

## Agenda January

   * Officially launch the Working group, identify:
   * - Chair: Esther
       * - Chairing meetings
       * - Liason between the WG and the constitutional levels, communicate decisions etc

>    * - Secretariat (other roles): will be split between Arielle and Emma

Reviewer: ```suggestion
   * Secretariat (other roles): will be split between Arielle and Emma
```

>        * - ensuring all the topics of the Agenda are covered

Reviewer: ```suggestion
       * Ensuring all the topics of the Agenda are covered
```

>        * - Notes are documented and uploaded in the repository (still need to decide if this should be an independent repository - we decided yes to the independent repo)

Reviewer: ```suggestion
       * Notes are documented and uploaded in the repository (still need to decide if this should be an independent repository - we decided yes to the independent repo)
```

>    * - invite other members (invitation link)

Reviewer: ```suggestion
   * Invite other members (invitation link)
```

>        * - Planning committee is self nominations

Reviewer: ```suggestion
       * Planning committee is self nomination from community members who have previously attended a Book Dash
```

>        * - WG is previous members of planning committee members as one of the key components

Reviewer: ```suggestion
       * WG has previous members of Book Dash planning committee as one of the key components
```

>        * - Aiming for 5 people in the WG

Reviewer: ```suggestion
       * Aiming for 5 people in the WG
```

>        * - List of previous planning committee members. Several of these people are already engaged in WGs and so we don't want to over load them with another WG. We also want more people not Turing staff as EK and AB are turing staff.

Reviewer: ```suggestion
       * List of previous planning committee members. Several of these people are already engaged in WGs and so we don't want to overload them with another WG. We also want more people not Turing staff as EK and AB are Turing staff.
```

>            * - Susana Roman-Garcia ---> To be asked

Reviewer: ```suggestion
           * Susana Roman-Garcia ---> To be asked
```

>            * - Carlos - may be too busy  ----> to be asked

Reviewer: ```suggestion
           * Carlos - may be too busy  ----> to be asked
```

>            * - Andrea Sanchez-Tapia: part of the Translation and localisation

Reviewer: ```suggestion
           * Andrea Sanchez-Tapia: part of the Translation and localisation
```

>            * - Alden Conner

Reviewer: ```suggestion
           * Alden Conner
```

>            * - Batool Almarzouq: part of Translation \& Localisation 

Reviewer: ```suggestion
           * Batool Almarzouq: part of Translation & Localisation WG
```

>            * - Johanna Bayer

Reviewer: ```suggestion
           * Johanna Bayer
```

>            * - Liz Hare: part of the Accessibility wg

Reviewer: ```suggestion
           * Liz Hare: part of the Accessibility WG
```

>            * - Saranjeet Kaur

Reviewer: ```suggestion
           * Saranjeet Kaur
```

>            * - Winny Nekesa

Reviewer: ```suggestion
           * Winny Nekesa
```

>            * - Alejandro? hasn't been a member of the planning committee

Reviewer: ```suggestion
           * Alejandro? Hasn't been a member of the planning committee
```

   * - Encourage men to apply to the planning committee
   * Agree on 2024 Book Dash dates: 

>    * 1st edition: 13-17 May - (TBA: timeline, forms below) / ALE will book rooms at The Turing and add it in the calendar

Reviewer: ```suggestion
   * 1st edition: 13-17 May - (TBA: timeline, forms below) / ALE will book rooms at The Turing and add it in the calendar [UPDATE - following a Slack discussion Book Dash is being held 3-6 June instead to accommodate annual leave from team members]
```

       * Arielle: earliest has to be 12 UK time
   * Next meeting: review application form for the participants as some of the questions are not friendly for new people
       * Info for the local hub needs to be in there
       * Time zone sessions confirmed so people know that it will be convenient for them
           * American timezone confirmed (Arielle)
           * Should ask Shern/Saranjeet for planning committee for earlier time zones
   * Book Dash Nov2023 report PR review: [https://github.com/the-turing-way/the-turing-way/pull/3503](https://github.com/the-turing-way/the-turing-way/pull/3503)
       * Should Kirstie/Malvika merge
       * Esther to check 
   * Transparency of their work - should we open a repo?
       * Infrastructure, accessibility, newsletter does have a repo
       * Have a bookdash repository for WG and planning committee
       * Emma/Ale to set it up
   * How they can allow others to see their work
   * Chapter in the Community Handbook
       * Need to include new information on the WG (next session / Feb)
   * Use of issue/discussing on main repoREADME, landing page (both in book and .github): 

### Timeline / Change dates to May 2024

>    * Call for application to join the Book Dash Planning Committee and local hub start date: 15 Feb 2024 (3 months before)

Reviewer: ```suggestion
   * Calls for applications to join the Book Dash Planning Committee and host local hubs open: 15 Feb 2024 (3 months before)
```

>    * Deadline to complete the Expression of Interest to Join *The Turing Way* Book Dash Committee: ~~31 March 2024 ~~1 March 2024 (plus community outreach)

Reviewer: ```suggestion
   * Deadline to complete the Expression of Interest to Join *The Turing Way* Book Dash Committee: 1 March 2024 (plus community outreach)
```

>    * Deadline for local hub - an expression of interest: ~~9 September 2023 ~~1 March 2024

Reviewer: ```suggestion
   * Deadline for local hub - an expression of interest: 1 March 2024
```

   * ALE WILL UPDATE THEM / suggestion

>    * Call for Applications for Participants: 24 July 2023

Reviewer: ```suggestion
   * Call for Applications for Participants: Tuesday 5 March 2024```

>    * Deadline for submission: 11 September 2023 (anywhere on Earth) - (originally on 15 September 2023 but extended)

Reviewer: ```suggestion
   * Deadline for submission: Monday 15 April (anywhere on Earth) ```

>    * Decisions on the applications: Latest by 01 October 2023

Reviewer: ```suggestion
   * Decisions on the applications: Latest by 01 May 2024
```

>    * Pre Book Dash Onboarding calls (2 x 1 hour): 30 October 2023. 

Reviewer: ```suggestion
   * Pre Book Dash Onboarding calls (2 x 1 hour): week beginning 22 May, exact date TBC
```

       * 9:00 am session: led Batool, Saranjeet, Emma, Anne, Alexandra
       * 17:00 session: Arielle, Anne, Alexandra
   * Pre Book Dash GitHub Skill-up (1 hour): May 2024, led by Batool Almarzouq? and Johanna Bayer?

>    * Book Dash Contribution Sessions during the week: of  13-16 May 2024

Reviewer: ```suggestion
   * Book Dash Contribution Sessions during the week: of 3-6 June 2024
```

>    * The Turing Way community share-out: 17 May 2024. Additions to The Turing Way Book

Reviewer: ```suggestion
   * The Turing Way community share-out: 7 June 2024. 
```

*Curated from the DAY-5 share out notes*

## Notes

* Idea of the WG is to be strategic, whereas the committee is operational
* We need more gender diversity in the planning committee so that the WG can then also become more diverse. 
* Candidates for community outreach for planning committee membership:

    - Richard Acton 

    - Alejandro Coca-Castro 

    - David Llewllyn-Jones

    - Jim Madge 

    - Kalle Westerling

    - Andrea Sanchez-Tapia

    - Alden Conner

    - Batool Almarzouq

    - Johanna Bayer

    - Liz Hare

    - Saranjeet Kaur

    - Winny Nekesa

###  Decisions / Action Items

   * Invite Susana and Carlos to join WG in first instance - Esther (done)
   * There is a need for a more balanced ratio of genders in the Book Dash planning committee
   * Community outreach to previous planning committee members and specific Book Dash attendees for the next planning committee applications - to be divided up 

>    * First Book Dash of 2024 will be 13-17 May 2024 

Reviewer 2: After some Slacks discussions, the agreed dates for the first Book Dash are 3-7 June 2024
